README.md

```diff
@@ -28,4 +28,10 @@ Install The following plugins
 
 Connect using jetpack. You will need a externally accessible url. You can use ngrok for this.
 
+If you want to set up an account with test data, set the `dev` mode flag to true. In your site's `wp-config.php`:
+
+```PHP
+define( 'WCPAY_DEV_MODE', true );
+```
+
```

Comment:
Hey @Aljullu good catch, and sorry for not keeping this up to date. However, there are public docs on that topic: https://docs.woocommerce.com/document/payments/testing/dev-mode/ with more details and context. What do you think of the following wording here:
```suggestion
For setting up a test account follow [these instructions](Hey @Aljullu good catch, and sorry for not keeping this up to date. However, there are public docs on that topic: https://docs.woocommerce.com/document/payments/testing/dev-mode/ with more details and context. What do you think of pointing ).
```

And since we're here it's worth noting the following:

1. Makes sense to put these lines before `Connect using jetpack...`.
2. `Jetpack` is not required anymore so it might be removed from the list.
3. Since `Jetpack` is not needed anymore worth updating the line `Connect using jetpack. ...` to something like this: `You will need a externally accessible URL to set up the plugin. You can use ngrok for this.`. 
4. `WooCommerce Admin` ships with WC Core, so it's safe to remove from the list as well.

Reply:
Ah, I wasn't aware of those docs. It makes a lot of sense linking them.

I updated the PR with your feedback, I couldn't directly commit your GH suggestion since it has some text from the code review mixed in between, instead I did the change myself (https://github.com/Automattic/woocommerce-payments/pull/987/commits/214633f40841a4b154a785de9cee3e8ae60ee671).

> let me know if you're fine adding new changes or I can do it myself

If you see something else that needs to be changed, feel free to commit directly if that's going to be faster. :slightly_smiling_face: